torch/fx/experimental/proxy_tensor.py

```diff
@@ -1110,6 +1110,35 @@ def create_node(
     ) -> torch.fx.Node:
         node = super().create_node(kind, target, args, kwargs, name, type_expr)  # type: ignore[arg-type]
 
+        # stack_trace
+        if "stack_trace" not in node.meta and node.op not in ["placeholder", "output"]:
```

Comment:
When would stack_trace already be in node.meta? This makes it a bit difficult to reason about this